src/openzaak/conf/locale/nl/LC_MESSAGES/django.po

```diff
@@ -6018,14 +6045,6 @@ msgstr ""
 msgid "Connection refused. Please provide a correct address."
 msgstr "Verbinding gewijgerd. Geef een geldig adres op."
 
-#: translations are missing from django-rest-framework
-msgid "Number of results to return per page."
-msgstr "Het aantal resultaten terug te geven per pagina."
-
-#: translations are missing from django-rest-framework
-msgid "A page number within the paginated result set."
-msgstr "Een pagina binnen de gepagineerde set resultaten."
-
```

Comment:
These translations were added to `commonground-api-common`, so they're no longer needed here